build.gradle

```diff
@@ -21,11 +18,11 @@ allprojects {
 }
 
 dependencies {
-    compile 'com.squareup.okhttp3:okhttp:3.2.0'
+    compile "com.squareup.okhttp3:okhttp:3.2.0"
     compile "org.slf4j:slf4j-api:1.7.7"
-    testRuntime "org.slf4j:slf4j-simple:1.7.7"
+    testRuntime "ch.qos.logback:logback-classic:1.1.3"
```

Comment:
made these strings more consistent. Swapped out log4j-simple for logback for more useful log statements in tests.